client/src/quiz/components/Quiz.js

```diff
@@ -4,6 +4,11 @@ import Question from '../components/Question';
 import QuestionCount from '../components/QuestionCount';
 import AnswerOption from '../components/AnswerOption';
 
+const getBackgroundColor=()=>{
```

Comment:
let's move this method to editpage.js so we can reuse it in the Result.js (eliminate code duplication). And import the function at the top of the page